src/index.ts

```diff
@@ -161,11 +163,16 @@ class Client {
   private constructor(
     private readonly clientConfig: AxiosRequestConfig,
     private readonly options: ConnectionOptions
-  ) {}
+  ) { }
 
   static create(options: ConnectionOptions): Client {
+    const agent = new https.Agent({
+      rejectUnauthorized: options.verifySSLCert
```

Comment:
Hi @elferherrera here you will need to do smth like 
```suggestion
      rejectUnauthorized: options.verifySSLCert ?? DEFAULT_VERIFY_SSL_CERT
```